tools/dpdk-setup-iface.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# check if there is only one additional command-line argument
+if [ $# -ne 4 ]
+then
+    echo "This script is used to setup ifconfig interfaces for dpdk"
```

Comment:
Make this "This script is used to setup a network adapter for a DPDK managed network interface.  If you run `ifconfig` after using this script, there will be an entry for a DPDK interface.  The interface will look similar to one managed by the kernel, but instead, this interface is managed by DPDK."